src/BlazorUI/Bit.BlazorUI/Components/Notifications/Tag/BitTag.scss

```
@@ -36,19 +169,57 @@
+    &:disabled {
+        cursor: default;
+        pointer-events: none;
+    }
 }
 
 
 .bit-tag-fil {
     color: var(--bit-tag-clr-txt);
     border-color: var(--bit-tag-clr);
     background-color: var(--bit-tag-clr);
+    // A filled tag is already wearing the role color, so it answers the pointer by deepening it. The overlay
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)